test/cookbooks/test/metadata.rb

```diff
@@ -7,3 +7,6 @@
 
 depends 'php'
 depends 'apt'
+depends 'ondrej_ppa_ubuntu'
+depends 'yum-epel'
+depends 'yum-remi-chef', '>= 5.0.1'
```

Comment:
What's the thinking behind moving these down into the test cookbook? 

The original thinking is that if they're needed for it to install, then they're cookbook deps, not test deps.

Reply:
@damacus I'm working with @ArtofBugs on this. The thinking is that the cookbook follow similar principles we have with other cookbooks:
1. Focus on making it work with the stock OS repos
2. Remove management of third-party from the cookbook, but make sure the new resource are flexible enough to deal with those third-party packages (thus why we moved them here).

Basically, the third-party repos are causing a lot of issues and really outside the scope of this cookbook.

Reply:
My thinking was that since these dependencies are only used by the `community` test recipe (the `php_install` resource itself doesn't specify where the packages are being installed from), they should be in the test metadata and not the general cookbook metadata. Would it make more sense to move them back?

Reply:
Oops, sorry for the redundant comment... my GitHub seems to be out of sync today for some reason...

Reply:
Yes, please. Our general thinking is that the cookbook should work off the shelf.

Reply:
@ramereth if that's our new direction then I'm happy to move that way 👍🏼 

Best document it over on sous-chefs.org or something similar. That way we can move a lot of our deps to the test cookbook all over the place.

Reply:
@damacus is the documentation on that site pulled in from the cookbooks automatically?

Reply:
Not sure, I can check and setup automation for it. 
mid you haven’t heard back by Monday from me ping me in slack